package.json

```diff
@@ -1,54 +1,55 @@
 {
   "name": "ember-cli-mousewheel",
   "version": "0.1.5",
```

Comment:
Please update the version.

Reply:
@systembugtj do you feel `0.1.6` would be fine? Since this is Ember CLI update does it make sense to bump up to 0.2.0?

Reply:
since it's using different approach, 0.2.0 is better. 👍

Reply:
Could you please update README also?

Reply:
I will bump the version.